Modules/GuessManager.cs

```diff
@@ -189,7 +189,7 @@ public static bool GuesserMsg(PlayerControl pc, string msg, bool isUI = false)
                 
                 if (Main.PlayerStates.TryGetValue(target.PlayerId, out var targetState) && targetState.Role != null)
                     if (targetState.Role.OnRoleGuess(isUI, target, pc))
-                        return true;
+                        return targetState.Role.OnRoleGuess(isUI, target, pc);
```

Comment:
There is no need to re-check since the `if (targetState.Role.OnRoleGuess(isUI, target, pc))` condition has already returned `true`

Reply:
idk how that got there, must be dark sorcery